platform-sdk/docs/proposals/TSS-Ledger-Id/TSS-Ledger-Id.md

```diff
@@ -298,6 +297,15 @@ Proof Sketch by Dr. Leemon Baird:
   shares.
 - So both theorems are proved.
 
+###### Threshold for Votes to Adopt a Candidate Roster
+
+The sum of the consensus weight of all the nodes that have voted yes to adopt the candidate roster must greater than or
+equal to 1/3 of the total consensus weight to ensure that at least one honest node has validated the TSS key
+material and voted yes.
+
+Since all votes are tabulated after reaching consensus and advancing consensus requires active nodes totalling 2/3
+consensus weight, the threshold of 1/2 consensus weight is safe and achievable.
```

Comment:
```suggestion
The sum of the consensus weight of all the nodes that have voted yes to adopt the candidate roster must be greater than or
equal to 1/3 of the total consensus weight to ensure that at least one honest node has validated the TSS key
material and voted yes.

Since all votes are tabulated after reaching consensus and advancing consensus requires active nodes totalling 2/3
consensus weight, the threshold of 1/2 consensus weight is safe and achievable.
```

In addition to the `be`, at line 303 you're talking `1/3`, and at line 307 you're talking `1/2`, in both the cases referring to it as a "consensus weight threshold". So I'm unsure if it's 1/3 or 1/2. And if it is 1/3, then I'm still unsure if it's 1/3 or (1/3+1), because 1/3 doesn't really seem sufficient to me.